### DISCUSSION TITLE
reset password

### DISCUSSION THREAD

**Discussion by jaakla** [lightdash/lightdash#1593] [Q&A] [answered]
I cannot login to my personal installation anymore, password manager somehow gives wrong password. Any way to reset or recover it?

**Comment by TuringLovesDeathMetal** [comment]
Hey @jaakla! Are you on a self-hosted or cloud version of Lightdash?

**Reply by TuringLovesDeathMetal** [reply]
ah, just re-read "personal installation", so I'm assuming that means self-hosted?

**Comment by ZeRego** [comment, marked as answer]
Hi @jaakla 

## How to override user password

1. Open bash terminal for the docker Lightdash container

2. Override user password with this command
```
cd ./packages/backend && node ./dist/overrideUserPassword.js <user email> <new password>
```

We recommended adding the [SMTP environment variables](https://docs.lightdash.com/references/environmentVariables#smtp-environment-variables) so Lightdash can display a “Recover forgotten password” button in the login page and send emails to reset password.

Let us know if these instructions are clear and if they worked :)

**Reply by TuringLovesDeathMetal** [reply]
Just updated the docs with these instructions!

you can find them here (in ~ 1hr when we re-deploy): https://docs.lightdash.com/guides/how-to-reset-your-password

:)
